resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/DepsTestsSuite.scala

```diff
@@ -63,7 +63,7 @@ private[spark] trait DepsTestsSuite { k8sSuite: KubernetesSuite =>
     ).asJava
 
     new ContainerBuilder()
-      .withImage("ceph/daemon:v4.0.0-stable-4.0-master-centos-7-x86_64")
+      .withImage("ceph/daemon:latest")
```

Comment:
this might break in a future (or future major) version of ceph/daemon? we are sure this will be always compatible?

Reply:
@felixcheung Nope but if it does I can ping the guys on ceph-nano to help. The other way is to push an image somewhere and use that. Is there a dockerhub repo which we can use?

Reply:
Unless Shane sets up a docker registry that we can use for the integration tests, I guess this is the best we can do for now. I'll merge this to get tests going, we can look at alternatives separately.